web-marketplace/src/lib/normalizers/creditClass/normalizeCreditClassOptions.types.ts

```diff
@@ -0,0 +1,8 @@
+export interface CreditClassOption {
```

Comment:
I realize this is reusing the naming that was already in place but is "options" the best way to describe this? Probably not something we need to address here but "params" or "info" might make more sense in this context.

Reply:
I think it was named "options" because in the "choose credit class" form, the credit classes are basically options of a select field.

Reply:
Good point, I checked and there doesn't seem to be a select there so I renamed it instead.
I renamed everything from `Options` to `Items` because these data are used by `ChooseCreditClassItem` component.

Reply:
Yeah it's not strictly a select, it just acts like that.
Renaming sounds good to me.